From 82b5436598848d6870174d04f1c3a5cf702c1508 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Wed, 11 Dec 2019 15:42:22 -0500
Subject: [PATCH] Issue 50780 - Fix UI issues

Description:  Fixed issue with replication conflict entry modal, and
              problem with the monitor's replication and suffix
              state data collision that caused a crash if you first
              looked at replication and then the suffix monitor

relates: https://pagure.io/389-ds-base/issue/50780

Reviewed by: spichugi(Thanks!)
---
 src/lib389/lib389/replica.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib389/lib389/replica.py b/src/lib389/lib389/replica.py
index 9b84d8f7e..4855f03f0 100644
--- a/src/lib389/lib389/replica.py
+++ b/src/lib389/lib389/replica.py
@@ -1169,7 +1169,7 @@ class Replica(DSLdapObject):
             if len(conflicts) > 0:
                 report = copy.deepcopy(DSREPLLE0002)
                 report['detail'] = report['detail'].replace('SUFFIX', suffix)
-                report['detail'] = report['detail'].replace('COUNT', len(conflicts))
+                report['detail'] = report['detail'].replace('COUNT', str(len(conflicts)))
                 report['fix'] = report['fix'].replace('YOUR_INSTANCE', self._instance.serverid)
                 yield report
 
-- 
2.24.1

